builtin/providers/google/resource_container_cluster.go

```diff
@@ -142,7 +142,56 @@ func resourceContainerCluster() *schema.Resource {
 				Default:  "default",
 				ForceNew: true,
 			},
-
+			"subnetwork": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"addons_config": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"http_load_balancing": &schema.Schema{
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							ForceNew: true,
```

Comment:
Since you set these as `Computed`, you have to set them in your `Read(..)` function at some point.